week-1-practise/level-2/3-currency-conversion.js

```diff
@@ -5,7 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(priceInGBP) {
+  return price * (1.29631).toFixed(2);
+}
 
```

Comment:
I'm curious as to why you've answered the question as you have - surely multiplying `price` by 1.4 would have got you the result you wanted. Also, did you test your solution before posting it? Did it give you the answer you expected?

Lastly, as per the comment on the previous question, what happens if this function is being used to convert currencies other than "pound" to $? Would the way the function is written still make sense, particularly when read by another programmer at a later date?